_includes/homepage_content.html

```diff
@@ -50,6 +50,17 @@ <h1 class="intro text-monospace display-1" style="margin-left: -.3rem;"><span cl
                                   </div>
                               </div>
 
+                              <div class="id-rpm64 hidden">
+                                  <div class="float-left">
+                                      <img src="{{ site.url }}/images/icon-linux-g.svg" class="os-icon os-icon-g">
+                                      <img src="{{ site.url }}/images/icon-linux-w.svg" class="os-icon os-icon-w">
```

Comment:
@pedromvpg any chance you would be able to create a red hat or fedora inspired image for this?